README.md

```diff
@@ -277,6 +277,36 @@ sys.addShutdownHook(system.terminate())
 ```
 
 ## Changelog  
+
+### 3.0.1 (2017-10-27)
+Changes since v3.0.0
+  - Akka 2.5 release only, please use the 2.7.x release for Akka 2.4 
+  - Align JournalTables with the sql schemas (issue #130)
+  - Current events by tag fix (issue #124)
+  - fixes for max ordering jump (issue #126)
+  - Allow configuration of ask timeout when querying maxOrdering (issue #123)
+  - Cache dependencies in Travis CI.
+  - sbt-release 1.0.6
+  - Akka 2.5.4 -> 2.5.6
+  - Merged PRs #122, #127, #128, #129, #132, #133, #135. Thanks to [Jimin Hsieh][jiminhsieh],
+    [Renato Cavalcanti][renatocaval], [Sean Sullivan][sullis],
+    [Ruud Welling][WellingR], [Adam Chaulk](acchaulk).
+
+
+### 2.7.1 (2017-10-27)
```

Comment:
Actually, since this release contains a breaking change, we should use 2.8.0 as the version number - it it not just a bugfix release anymore